_parts/part03.md

```diff
@@ -129,19 +132,20 @@ Let's work through the example following the steps
 
-In our example, $$0+6=6$$ and since $$6<10$$, we mark the first digit of our result (i.e the right-most digit also known as the *least significant digit*) as 6. Then, we consider the next digit position and add $$5+8=13$$. Since $$13>10$$, we let the position of the result by $$13-10=3$$ and then note the carry over. When we look at the next digit position, we add $$7+9+1=17$$ (the 1 appears from the carry over of the previous digit position). Since $$17>10$$ we let the corresponding digit position of our result by $$17-10=7$$. Now, although there are no more digit positions left to sum, our final (most significant) digit position, had a carry over, so we must account for that. You can think of this as one final sum, where the two values are 0, as shown in the table below. In that case we add $$0+0+1=1$$ which means we have a $$1$$ in the left-most digit position of our result. It is safe to prepend these 'fake' zeroes to our numbers under addition because adding zeroes to the front of a number does not change its intrinsic value.
+Let's break down our initial example from the paragprah above $$780+956$$. In the one's place, we have $$0+6=6$$; since $$6<10$$, we mark the first digit of our result (i.e the right-most digit also known as the **least significant digit**) as 6. Then, we consider the next digit position and add $$5+8=13$$. Since $$13>10$$, we let the position of the result by $$13-10=3$$ and then note the carry over. When we look at the next digit position, we add $$7+9+1=17$$ (the 1 appears from the carry over of the previous digit position). Since $$17>10$$ we let the corresponding digit position of our result by $$17-10=7$$. Now, although there are no more digit positions left to sum, our final (most significant) digit position, had a carry over, so we must account for that. You can think of this as one final sum, where the two values are 0, as shown in the table below. In that case we add $$0+0+1=1$$ which means we have a $$1$$ in the left-most digit position of our result. It is safe to prepend these 'fake' zeroes to our numbers because adding zeroes to the front of a number does not change its intrinsic value.
 
 | Computing X+Y  |   |   |   |   |
 |---|---|---|---|---|
+| Carry-Over | 1 | 1 | 1 | |
```

Comment:
What is the meaning of this added line; How does it differ from what is on line 149?